ccd-definition/ComplexTypes/SolicitorReferences.json

```
@@ -18,5 +18,16 @@
     "Max": "24",
     "_Definition":  "The reference value understood by the first respondent solicitor firm for the [claim / respondent]",
     "_Category":  "COMMON"
+  },
+  {
+    "ID": "SolicitorReferences",
+    "ListElementCode": "respondentSolicitor2Reference",
+    "FieldType": "Text",
+    "ElementLabel": "Defendant's legal representative's reference",
+    "SecurityClassification": "Public",
+    "Max": "24",
+    "_Definition":  "The reference value understood by the first respondent solicitor firm for the [claim / respondent]",
+    "FieldShowCondition": "respondentSolicitor1Reference=\"DO_NOT_SHOW_IN_UI\"",
```

Should this be `respondentSolicitor2Reference` ?
```suggestion
    "FieldShowCondition": "respondentSolicitor2Reference =\"DO_NOT_SHOW_IN_UI\"",
```